src/main/java/create_task_use_case/CreateTaskController.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Create Task Controller Class.
+ * Author: Fardin Faruk
+ * Modified By: Fardin Faruk
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 27, 2022
+ */
+package create_task_use_case;
+
+public class CreateTaskController {
+
+    private final CreateTaskInputBoundary inputBoundary;
+
+    public CreateTaskController(CreateTaskInputBoundary inputBoundary) {
+        this.inputBoundary = inputBoundary;
+    }
+
+    CreateTaskOutputData createTask(String title, boolean completed, int dayIndex){
```

Comment:
Why does it take in a completion status? Can we assume that the task will be incomplete by default upon creation?

Reply:
Good catch, I fixed it up.